pkg/types/core/median.go

```diff
@@ -0,0 +1,14 @@
+package core
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2/reportingplugin/median"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+)
+
+type PluginMedian interface {
```

Comment:
why is this a core type? it seems out of place relative to the other things

Reply:
It's analogous to the reporting plugin factories methods, except median-specific (dating from when median was a custom loopp type.